strongbox-security/strongbox-user-management/src/test/java/org/carlspring/strongbox/repositories/UserRepositoryTest.java

```diff
@@ -0,0 +1,91 @@
+package org.carlspring.strongbox.repositories;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import org.carlspring.strongbox.config.DataServiceConfig;
+import org.carlspring.strongbox.config.UsersConfig;
+import org.carlspring.strongbox.domain.User;
+import org.carlspring.strongbox.domain.UserEntity;
+import org.carlspring.strongbox.gremlin.adapters.EntityTraversalUtils;
+import org.carlspring.strongbox.users.domain.SystemRole;
+
+import javax.inject.Inject;
+
+import java.util.Date;
+import java.util.List;
+import java.util.Set;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.ContextConfiguration;
+
+import com.google.common.collect.Sets;
+
+/**
+ * @author ankit.tomar
+ */
+@SpringBootTest
+@ActiveProfiles(profiles = "test")
+@ContextConfiguration(classes = { DataServiceConfig.class,
```

Comment:
```suggestion
@ContextConfiguration(classes = RepositoriesTestConfig.class)
```

Reply:
Done.

Reply:
This change is breaking other tests. Also `RepositoriesTestConfig` is itself importing `DataServiceConfig.class`.